src/setupTests.js

```diff
@@ -0,0 +1,8 @@
+import { configure } from 'enzyme'
+import Adapter from 'enzyme-adapter-react-16'
+
+configure({ adapter: new Adapter() })
+
+beforeEach(() => {
+  jest.clearAllMocks()
+})
```

Comment:
Automatically clears all of our mocks before each test. This is needed so we can check if a mock is called once, confirming we didn't accidentally trigger re-renders or similar.

Jest has three levels of clearing mocks:
- [clear](https://jestjs.io/docs/en/mock-function-api#mockfnmockclear) - resets call counts
- [reset](https://jestjs.io/docs/en/mock-function-api#mockfnmockreset) - clear + reset return values
- [restore](https://jestjs.io/docs/en/mock-function-api#mockfnmockrestore) - reset + restores original functionality

Reply:
This file `src/setupTests.js` will be our hook into adding testing configurations and setup for our Jest environment. We can add what we need without having to eject our app.